wikipedia-dark.user.css

```diff
@@ -2853,6 +2853,7 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
   .mw-echo-ui-notificationBadgeButtonPopupWidget-popup > .oo-ui-popupWidget-popup > .oo-ui-popupWidget-head,
   .mw-echo-ui-notificationItemWidget:last-child {
     border-color: var(--gray-5);
+    background-color: var(--gray-2);
```

Comment:
I would instead be more specific and add bg color to the specific selector needed instead of all rule.

It will minimize the chances of breaking N over fixing x

Reply:
Oops, seems like I made a mistake here, I was meant for the selector below this. As you can see in line 2867 here: https://github.com/StylishThemes/Wikipedia-Dark/pull/162/files/e04f567f362d6a8e7f551e516b47acee2d4b9587#diff-b52ab5bfdb3a915b5378cc6e1f512df5667751a1d00a9e55ef613eac36ef46a7R2866, the selector already contains `html .thumbimage`. So my intention was to move the deleted one from https://github.com/StylishThemes/Wikipedia-Dark/pull/162/files/e04f567f362d6a8e7f551e516b47acee2d4b9587#r565533042 into here. Let me fix it

Reply:
Ok, let me fix it